dataeng/resources/warehouse-transforms.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/warehouse-transforms
+pip install -r tools/dbt_schema_builder/requirements.txt
```

Comment:
nit: suggesting adding a comment here indicating that this is used to install the right version of dbt.